lib/pfconfig/manager.pm

```diff
@@ -131,21 +131,7 @@ sub touch_cache {
     my $filename = pfconfig::util::control_file_path($what);
     $filename = untaint_chain($filename);
 
-    if ( !-e $filename ) {
-        my $fh;
-        unless ( open( $fh, ">$filename" ) ) {
-            $logger->error("Can't create $filename\nPlease run 'pfcmd fixpermissions'");
-            return 0;
-        }
-        close($fh);
-    }
-    if ( -e $filename ) {
-        sysopen( my $fh, $filename, O_RDWR | O_CREAT );
-        POSIX::2008::futimens( fileno $fh );
-        close($fh);
-    }
-    my ( undef, undef, $uid, $gid ) = getpwnam('pf');
-    chown( $uid, $gid, $filename );
+    `touch $filename`;
```

Comment:
This is an old commit you should get rid of touching the file use the touch binary